### ISSUE TITLE
Study Feedback: Family-Facing Aspects

### ISSUE THREAD

**Issue by msheskin** [lookit/lookit-api#963]
# TL;DR
After a family participates in a study, they see a final screen (before being dropped into their study history) with optional questions about the study they just finished.

# Narrative
As a parent who has just invested time into participating in a research study, I would like an opportunity to report back to the Lookit team the overall quality of the study and any thoughts I have about it. This allows me to express appreciation for great studies, and concerns about studies I might find problematic. 

# Implementation Notes
We should probably workshop the specific ideas among Mark/CJ/Melissa. For example, we could phrase it as being information ONLY to Lookit, or to BOTH Lookit and the researchers who did the study, or have multiple fields for both. As a start, imagine something like:

This final page is from the Lookit team at MIT. Thank you so much for participating in one of the studies from researchers using Lookit! This optional page allows you to provide feedback about the study your family just finished. Your answers to these optional questions allows us to understand which types of studies families are enjoying the most, and also can help us identify any studies that should be changed or improved in a specific way. 

Optional: Rate the study you just saw on a scale from 1 to 5
1: This study should not be on Lookit
2: This study was fine, but I would not recommend it to a friend
3: This study was good (about what I would expect on Lookit)
4: This study was great, and I would recommend it to a friend
5: This study was amazing!

Optional: Provide the Lookit team at MIT more information about your rating, or any other thoughts about the study
[text box]
